server/sequelize/sequelize.config.ts

```diff
@@ -0,0 +1,11 @@
+export const Configuration = {
+    development: {
```

Comment:
Add test config as it is in develop. Remove old config

Reply:
Should i add production as well as it is in develop

Reply:
For now - doesn't matter. Do if production in develop is really used